util/build.gradle.kts

```diff
@@ -0,0 +1,248 @@
+            credentials {
+                username = (project.findProperty("gpr.user")
+                    ?: System.getenv("PACKAGE_REGISTRY_USERNAME")).toString()
+                password = (project.findProperty("gpr.key")
+                    ?: System.getenv("PACKAGE_REGISTRY_TOKEN")).toString()
+            }
+        }
+
+        val target = "file://${project.buildDir}/gitPublish"
+
+        maven {
+            name = "ReleasePackages"
+            url = uri("$target/releases")
+        }
+
+        maven {
+            name = "SnapshotPackages"
+            url = uri("$target/snapshots")
+        }
+
+        maven {
+            name = "FeaturePackages"
+            url = uri("$target/features")
+        }
+    }
+}
+
+configure<care.data4life.gradle.git.publish.GitPublishExtension> {
+    repoUri.set("git@github.com:d4l-data4life/maven-repository.git")
+
+    branch.set("main")
+
+    contents {
+    }
+
+    preserve {
+        include("**/*")
+    }
+
+    commitMessage.set("Publish ${LibraryConfig.name} $version")
+}
+
+val publishFeature by tasks.creating {
+    group = "publishing"
+
+    dependsOn("gitPublishReset")
+    dependsOn("gitPublishCopy")
+    dependsOn("publishAllPublicationsToFeaturePackagesRepository")
+    dependsOn("gitPublishCommit")
+    dependsOn("gitPublishPush")
+
+    tasks.findByName("gitPublishCopy")!!.mustRunAfter("gitPublishReset")
+    tasks.findByName("publishAllPublicationsToFeaturePackagesRepository")!!.mustRunAfter("gitPublishCopy")
+    tasks.findByName("gitPublishCommit")!!.mustRunAfter("publishAllPublicationsToFeaturePackagesRepository")
+    tasks.findByName("gitPublishPush")!!.mustRunAfter("gitPublishCommit")
+}
+
+val publishSnapshot by tasks.creating {
+    group = "publishing"
+
+    dependsOn("gitPublishReset")
+    dependsOn("gitPublishCopy")
+    dependsOn("publishAllPublicationsToSnapshotPackagesRepository")
+    dependsOn("gitPublishCommit")
+    dependsOn("gitPublishPush")
+
+    tasks.findByName("gitPublishCopy")!!.mustRunAfter("gitPublishReset")
+    tasks.findByName("publishAllPublicationsToSnapshotPackagesRepository")!!.mustRunAfter("gitPublishCopy")
+    tasks.findByName("gitPublishCommit")!!.mustRunAfter("publishAllPublicationsToSnapshotPackagesRepository")
+    tasks.findByName("gitPublishPush")!!.mustRunAfter("gitPublishCommit")
+}
+
+val publishRelease by tasks.creating {
+    group = "publishing"
+
+    dependsOn("gitPublishReset")
+    dependsOn("gitPublishCopy")
+    dependsOn("publishAllPublicationsToReleasePackagesRepository")
+    dependsOn("gitPublishCommit")
+    dependsOn("gitPublishPush")
+
+    tasks.findByName("gitPublishCopy")!!.mustRunAfter("gitPublishReset")
+    tasks.findByName("publishAllPublicationsToReleasePackagesRepository")!!.mustRunAfter("gitPublishCopy")
+    tasks.findByName("gitPublishCommit")!!.mustRunAfter("publishAllPublicationsToReleasePackagesRepository")
+    tasks.findByName("gitPublishPush")!!.mustRunAfter("gitPublishCommit")
+}
```

Comment:
Since you already touched it - do you plan to align it with the way of the SDK?

Reply:
yes, I could add it directyl

Reply:
it's replaced